pkg/sdn/plugin/proxy.go

```diff
@@ -78,7 +79,9 @@ func (proxy *ovsProxyPlugin) watchEgressNetworkPolicies() {
 			policy.Spec.Egress = nil
 		}
 		proxy.updateNetworkPolicy(*policy)
-		// FIXME: poke the endpoint-syncer somehow...
+		if proxy.lastEndpoints != nil {
+			proxy.OnEndpointsUpdate(proxy.lastEndpoints)
```

Comment:
there's still a race condition here that could make us lose the list of endpoints:

* A - `watchEgressNetworkPolicies` sees a non-nil `lastEndpoints`, calls `OnEndpointsUpdate(lastEndpoints)`
* B - syncer calls `OnEndpointsUpdate()` with a new list of endpoints
* B wins the race, sets `proxy.lastEndpoints` to the new list
* then A comes in and overwrites `proxy.lastEndpoints` with a stale list